lms/djangoapps/certificates/views.py

```diff
@@ -330,7 +338,7 @@ def _update_certificate_context(context, course, user, user_certificate):
         year=user_certificate.modified_date.year
     )
 
-    accd_course_org_html = '<span class="detail--xuniversity">{partner_name}</span>'.format(partner_name=course.org)
+    accd_course_org_html = '<span class="detail--xuniversity">{partner_name}</span>'.format(partner_name=partner_name)
```

Comment:
I'll see if I can clean this up -- some of these variables are holdovers from the initial implementation and are now obsolete -- maybe this is one of them.